include/thread.h

```diff
@@ -41,6 +42,10 @@ typedef struct thread {
   /* scheduler part */
   td_prio_t td_prio;
   int td_slice;
+  /* sbrk support */
```

Comment:
`vm_map` is a place to keep this information. Note, that several threads can share one `vm_map` instance.